client/service_inline.go

```diff
@@ -65,7 +65,7 @@ type serviceInlineClient struct {
 type ServerInitialInfo interface {
 	Endpoints() endpoint.Endpoint
 	Option() *internal_server.Options
-	GetServiceInfo() *serviceinfo.ServiceInfo
+	GetServiceInfo() *serviceinfo.Services
```

Comment:
I think returning map[string]*serviceinfo.ServiceInfo is better, I prefer to make Services and Service invisible to users